### PR TITLE
User creation form plus associated api method

### PR DESCRIPTION
**Pull request by dsilberger**

### CODE REVIEW

File: client/src/components/UserForm.jsx [hunk 1]
```diff
@@ -20,7 +20,14 @@ class UserForm extends React.Component {
   }
 
   handleClick(event) {
-    // model from Joe
+    event.preventDefault();
+    let newUserObject = {
+      name: this.state.username,
+      income: Number(this.state.income),
```
Comment (chengmuy) [inline client/src/components/UserForm.jsx]: Shouldn't this be `this.state.income * 100`?

File: client/src/api.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ const api = {
   fetchAllExpenses: () => {
     const config = { url: "/expenses" };
     return axios(config).then(({ data: expenseList }) => expenseList);
+  },
+  handleUserSubmit: data => {
+    console.log(`Called with`, data);
```
Comment (chengmuy) [inline client/src/api.js]: Can this console.log be removed?

File: client/src/components/UserForm.jsx (hunk 1 above)
Comment (dsilberger) [reply client/src/components/UserForm.jsx]: Agreed

File: client/src/api.js (hunk 2 above)
Comment (dsilberger) [reply client/src/api.js]: Yes

**Pull request merged**